src/sentry/api/endpoints/issues/related_issues.py

```diff
@@ -10,6 +11,12 @@
 from sentry.types.ratelimit import RateLimit, RateLimitCategory
 
 
+class RequestSerializer(serializers.Serializer):
+    type = serializers.CharField(required=True)
```

Comment:
I think you can use `ChoiceField` here as well to constrain it to the particular algorithm names

Reply:
thank you!